### PR TITLE
[docs]: fix documentation for the Steps Component 

### PR DESCRIPTION
**Pull request by prathamesh424**

### CODE REVIEW

File: docs/docs/widgets/steps.md [hunk 1]
```diff
@@ -64,41 +70,41 @@ There are currently no CSA (Component-Specific Actions) implemented to regulate
...
+<div style={{paddingTop:'24px'}}>
 
 ## Layout
 
 | <div style={{ width:"100px"}}> Layout </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
 |:--------------- |:----------------------------------------- | :------------------------------------------------------------------------------------------------------------- |
-| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
-| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
+| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on `fx` to set the value `{{true}}` or `{{false}}` |
+| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on `fx` to set the value `{{true}}` or `{{false}}` |
```
Comment (RudraPratik30) [inline docs/docs/widgets/steps.md:91]: Hey @prathamesh424, fx should be in bold and not enclosed in the back tick, please change it in the whole doc and both versions wherever applicable.

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @prathamesh424, please make the mentioned changes.
Thanks!

**Comment by prathamesh424** [comment]
@RudraPratik30  , @PriteshKiri  fixing `fx` -> **fx**

File: docs/docs/widgets/steps.md [hunk 2]
```diff
@@ -36,25 +38,29 @@ This property when disabled will disable selection of steps.
...
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Events
 
 | <div style={{ width:"100px"}}> Event </div> | <div style={{ width:"100px"}}> Description </div> |
 |:------------------|:---------------------|
 | On select | This event is fired whenever the user selects any step. |
+| On change | This event triggers when the user changes the text in the input. |
```
Comment (RudraPratik30) [inline docs/docs/widgets/steps.md:48]: Hey @prathamesh424, It looks like you have directly copied the events table from the text input component while you were just expected to refer it and update the content according to the steps component and please check if this events are even available for step component in tooljet app builder.

Please update this table in both versions.

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]

File: docs/docs/widgets/steps.md (hunk 2 above)
Comment (prathamesh424) [reply docs/docs/widgets/steps.md:48]: where I can find the events  so will add containtent based on that

Comment (PriteshKiri) [reply docs/docs/widgets/steps.md:48]: Hey @prathamesh424,

You just need to update the description of the "on select" event. The reference is provided to help you understand the sentence structure used in other components.

**Comment by prathamesh424** [comment]
@RudraPratik30 done

**Review by RudraPratik30 — APPROVED** [review]
Hey, @prathamesh424 approving the changes from my end, and forwarding it to the team for final review.
Thanks for contributing!
Your PR will be merged by the end of this week.

**Comment by RudraPratik30** [comment]
Hey @prathamesh424 can you please sign the [Contributor License Agreement](https://cla-assistant.io/ToolJet/ToolJet?pullRequest=10905) before we can accept your contribution.

**Comment by prathamesh424** [comment]
@RudraPratik30 signed the contributor license

**Review by amanregu — APPROVED** [review]

**Pull request merged**